cfgov/alerts/github_alert.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+import github3
+
+
+class GithubAlert(object):
```

Comment:
Putting on my @Scotchester hat to make a super-nitpicky observation that this should be `GitHubAlert` even though I always type "Github" every time I refer to the company that makes the product I am typing on right tnow.